app/assets/stylesheets/application.scss

```diff
@@ -31,6 +31,7 @@ $govuk-assets-path: "/";
 @import "table-sort";
 @import "tables";
 @import "sort-transactions";
+@import "summary-card";
```

Comment:
Import order: are we certain this import order is correct?

Reply:
Where would you think it should go? At the end? I must admit, i put it in kind of at random

Reply:
For now, the order works, based on the class in this new file.
We need to rethink the css architecture to ensure this doesnt become a problem.